fix: [MTM-58369] add missing release notes for UI (#1870)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-10-18-499-4-smart-rules-tab-view-sets-incorrect-breadcrumbs.md

---
date: 

> title: Smart rules tab view sets incorrect breadcrumbs in device detail view

Reviewer: Below it says that this is in the groups view?
```suggestion
title: Breadcrumb for the Smart rules tab correctly set in the device details view
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Platform services
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-51262
version: 10.18.499.4
---

> There is no breadcrumbs for Global smart rules in group view. If package.json is configured in a way to show breadcrumbs, then breadcrumbs will be shown in proper way.

Reviewer: ```suggestion
In the Groups view, the breadcrumb for the Smart rules tab was missing. If the *package.json* is configured in a way to show breadcrumbs, the breadcrumb will now be properly shown.
```

> There is no breadcrumbs for Global smart rules in group view. If package.json is configured in a way to show breadcrumbs, then breadcrumbs will be shown in proper way.

Author: we definitely need to remove `package.json` from here, so I tend to agree. Can we just replace it with term of `Application options`?

> In the Groups view, the breadcrumb for the Smart rules tab was missing. If the *Application options* are configured in a way to show breadcrumbs, the breadcrumb will now be properly shown.

Reviewer: ```suggestion
In the Groups view, the breadcrumb for the Smart rules tab was missing. If the *Application options* are configured in a way to show breadcrumbs, the breadcrumb will now be shown properly.
```
